src/routes/safe/container/hooks/useLoadSafe.tsx

```diff
@@ -16,12 +16,12 @@ export const useLoadSafe = (safeAddress) => {
       if (safeAddress) {
         dispatch(fetchLatestMasterContractVersion())
           .then(() => dispatch(fetchSafe(safeAddress)))
+          .then(() => dispatch(fetchSafeTokens(safeAddress)))
```

Comment:
why can't we fetch safe and safe tokens simultaneously?

Reply:
Probably we can, I was just focused on the need for `knownTokens` before fetching txs and didn't want to break anything.